bundles/io/org.eclipse.smarthome.io.rest.core/doc/RESTimprovements.md

```diff
@@ -0,0 +1,153 @@
+# non-breaking REST API changes
```

Comment:
please remove this file from the sources - it is great to have this on the description of the PR, though.